ddtrace/span.py

```diff
@@ -283,8 +283,12 @@ def get_metric(self, key):
         return self.metrics.get(key)
 
     def to_dict(self):
+        trace_id = self.trace_id
+        # ensure 128 bit trace IDs are always trimmed
+        if self.trace_id.bit_length() > 63:
```

Comment:
Is this check really necessary? Can't we just do the operation below for any `trace_id`?

Reply:
we could, was trying to avoid unnecessary formatting